docs/get-started/how-it-works.md

```diff
@@ -36,7 +36,7 @@ From an engineering perspective,
 we have hidden all the complexity of machine learning algorithms and resource configuration (such as GPUs).
 All you need to do is decide on your backbone model and prepare your training data.
 
-Once you logged into the Jina Ecosystem with {meth}`~finetuner.login()`, 
+Once you logged into the Jina Ecosystem with {meth}`~finetuner.login()` or `~finetuner.notebook_login()`, 
```

Comment:
```suggestion
Once you have logged in to the Jina Ecosystem with {meth}`~finetuner.login()` or `~finetuner.notebook_login()`, 
```

### Usage examples

- I will log in to foo
- I am logged in
- My login for foo is bar

God, I hate [grammar](https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to) :sweat_smile: